### PR TITLE
Bump to 1.1.0rc1

### PR DESCRIPTION
**Pull request by VVsssssk**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Bump to 1.1.0rc1

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/notes/changelog.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
 # Changelog of v1.1
 
+### v1.1.0rc1 (9/10/2022)
+
+#### New Features
+
+- Refactor PointRCNN in mmdet3d 1.1.x version (#1819)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:7]: Refactor PointRCNN in mmdet3d 1.1 version

File: docs/en/notes/changelog.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+
+#### Improvements
+
+- Add the code of generating cam_sync_labels in waymo dataset (#1870)
+- Update SUN RGB-D and Lyft datasets docs (#1833)
+- Update customized dataset docs (#1845)
+- Update dataset transforms typehints (#1875)
+- Fix ci (#1813, #1865, #1877)
+- Update browse dataset script (#1817)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:16]: `browse_dataset.py`

File: docs/en/notes/changelog.md [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+- Fix convert_to_datasample interface in detector (#1843)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:17]: raname `convert_to_datasample` to `add_pred_to_datasample` in detectors

File: docs/en/notes/changelog.md [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+- Add auto_scale_lr in config (#1807)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:18]: Add auto_scale_lr in config to support auto-scale learning rate training

File: docs/en/notes/changelog.md [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+
+#### Bug Fixes
+
+- Fix nuscenes dataset lack of num_pts_feats key (#1882)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:22]: Fix the lack of `num_pts_feats` key in nuscenes dataset and complete  docstring

File: docs/en/notes/changelog.md [hunk 6]
```diff
@@ -1,5 +1,38 @@
...
+- Fix mono3d detection demo (#1864)
+- Fix visualization bug (#1828, #1839)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:24]: we need to specify the bug detail

File: docs/en/notes/changelog.md [hunk 7]
```diff
@@ -1,5 +1,38 @@
...
+- Fix dbsampler bug (#1812)
```
Comment (ZCMax) [inline docs/en/notes/changelog.md:25]: Fix the dbsampler bug to use ground plane data

File: docs/en/notes/changelog.md [hunk 8]
```diff
@@ -1,5 +1,38 @@
 # Changelog of v1.1
 
+### v1.1.0rc1 (9/10/2022)
```
Comment (Tai-Wang) [inline docs/en/notes/changelog.md:3]: 11/10/2022

File: docs/en/notes/changelog_v1.0.x.md [hunk 9]
```diff
@@ -1,5 +1,29 @@
 # Changelog of v1.0.x
 
+### v1.0.0rc5 (9/10/2022)
```
Comment (Tai-Wang) [inline docs/en/notes/changelog_v1.0.x.md:3]: 11/10/2022

File: docs/en/notes/changelog_v1.0.x.md [hunk 10]
```diff
@@ -1,5 +1,29 @@
...
+
+#### New Features
+
+- Add indoor monocular 3d object detector ImVoxelNet on SUN RGB-D (#1738)
```
Comment (Tai-Wang) [inline docs/en/notes/changelog_v1.0.x.md:7]: Adjust this changelog according to comments of another release PR

File: docs/en/notes/changelog.md [hunk 11]
```diff
@@ -1,5 +1,38 @@
...
+
+#### New Features
+
+- Refactor PointRCNN in mmdet3d 1.1 version (#1819)
```
Comment (Tai-Wang) [inline docs/en/notes/changelog.md:7]: Refactor PointRCNN in the framework of mmdet3d v1.1

File: docs/en/notes/faq.md [hunk 12]
```diff
@@ -6,14 +6,15 @@ We list some potential troubles encountered by users and developers, along with
...
+- The required versions of MMEngine, MMCV and MMDetection for different versions of MMDetection3D are as below. Please install the correct version of MMEngine, MMCV and MMDetection to avoid installation issues.
 
-  | MMDetection3D version |     MMEngine version     |      MMCV version       |   MMDetection version    |
-  | :-------------------: | :----------------------: | :---------------------: | :----------------------: |
-  |        dev-1.x        | mmengine>=0.1.0, \<0.2.0 | mmcv>=2.0.0rc0, \<2.1.0 | mmdet>=3.0.0rc0, \<3.1.0 |
-  |       v1.1.0rc0       | mmengine>=0.1.0, \<0.2.0 | mmcv>=2.0.0rc0, \<2.1.0 | mmdet>=3.0.0rc0, \<3.1.0 |
+| MMDetection3D version |     MMEngine version     |        MMCV version        |     MMDetection version     |
+| --------------------- | :----------------------: | :------------------------: | :-------------------------: |
+| dev-1.x               | mmengine>=0.1.0, \<0.2.0 | mmcv>=2.0.0rc0, \<2.0.0rc2 | mmdet>=3.0.0rc0, \<3.0.0rc2 |
+| v1.1.0rc1             | mmengine>=0.1.0, \<0.2.0 | mmcv>=2.0.0rc0, \<2.0.0rc2 | mmdet>=3.0.0rc0, \<3.0.0rc2 |
```
Comment (Tai-Wang) [inline docs/en/notes/faq.md:14]: Please keep the original version constraints

File: README.md [hunk 13]
```diff
@@ -82,9 +82,9 @@ This project is released under the [Apache 2.0 license](LICENSE).
 
 ## Changelog
 
-We are excited to announce the release of MMDetection3D 1.1.0rc0.
+**1.1.0rc1** was released in 11/10/2022.
 
-MMDet3D 1.1 unifies the interfaces of dataset, models, evaluation, and visualization with faster training and testing speed.
+- We released `waymo 3D camera-only detection challenge` solution with evaluation metrics, transformation, and `DFM` models in 1.1.0rc1 version.
```
Comment (ZCMax) [inline README.md:87]: Since we add new models, we should add it in the table of README.md

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**